aldryn_people/boilerplates/bootstrap3/templates/aldryn_people/includes/group.html

```diff
@@ -0,0 +1,41 @@
+{% load i18n aldryn_people_tags %}
+
+<h2>{{ group.name }}</h2>
+
+<div class="content">
+    {{ group.description|safe }}
+</div>
+
+<div class="row">
+    <div class="meta
```

Comment:
we use generic names within a given namespace. There is already the `aldryn-people-list` or `aldryn-people-list-group` namespace wrapped. Thats the case for all essential addons